bddl/activity_definitions/installing_appliances/problem0.bddl

```diff
@@ -13,9 +13,9 @@
     )
     
     (:init 
-        (attached hand_tool.n.01_1 sink.n.01_1) 
-        (attached hand_tool.n.01_1 mirror.n.01_1) 
-        (attached hand_tool.n.01_1 door.n.01_1) 
+        (attached screwdriver.n.01_1 sink.n.01_1) 
+        (attached screwdriver.n.01_1 mirror.n.01_1) 
+        (attached screwdriver.n.01_1 door.n.01_1) 
```

Comment:
these initial conditions look a bit dubious..

Reply:
Yeah I was confused about that too - before, it was `attached hand_tool.n.01`. I'll just change it to something else.